packages/sdk-react-native/src/env.ts

```diff
@@ -75,22 +76,35 @@ export const EnvVars = {
 };
 
 export const generateEnvVarsFile = async () => {
+    const c = getContext();
+    if (!c.platform) return;
+
+    const isApplePlatform = c.platform === 'ios' || c.platform === 'tvos';
+    const fileName = isApplePlatform ? '.xcode.env.local' : '.env';
     const destDir = getAppFolder();
-    const destPath = path.join(destDir, '.env');
+    const destPath = path.join(destDir, fileName);
+
     const envVars: Record<string, any> = {
         ...CoreEnvVars.BASE(),
-        RNV_EXTENSIONS: CoreEnvVars.RNV_EXTENSIONS().RNV_EXTENSIONS.join(', '),
+        RNV_EXTENSIONS: CoreEnvVars.RNV_EXTENSIONS().RNV_EXTENSIONS.join(','),
         ...EnvVars.RCT_METRO_PORT(),
         ...EnvVars.RNV_REACT_NATIVE_PATH(),
         ...EnvVars.RCT_NO_LAUNCH_PACKAGER(),
         ...EnvVars.RNV_APP_ID(),
         ...EnvVars.RCT_NEW_ARCH_ENABLED(),
         ...EnvVars.RNV_FLIPPER_ENABLED(),
         ...EnvVars.RNV_SKIP_LINKING(),
+        ...(isApplePlatform
+            ? {
+                  NODE_BINARY: await executeAsync(`which node`, {
+                      cwd: c.paths.project.dir,
+                  }),
+              }
+            : {}),
     };
     let env = '';
     Object.keys(envVars).forEach((key) => {
-        env += `${key}=${envVars[key]}\n`;
+        env += ` ${isApplePlatform ? 'export' : ''} ${key}=${envVars[key]}\n`;
```

Comment:
A bit strange that it worked before without this if it's needed

Reply:
Previously it worked because we only generated .env file which didn't need `export`. Now we generate two types of files. .enf for android and xcode.env.local for apple platforms